content/guides/livecontainers.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Datadog Process and Container Monitoring
+kind: guide
+listorder: 16
+beta: true
+---
+
+## Introduction
+
+Datadog Live Containers allows for real-time visibility of all of the containers across your environment.  Taking inspiration from bedrock tools like `htop` and `ctop`, this centralized view, combined with existing tagging capabilities, allows you to understand what is going on at any level of your system and drill all the way down into the most fine details.
+
+{{< img src="process/live-container-table-flow.gif" >}} 
+
+## Searching, Filtering, and Pivoting
+
+### String Search
+
+Containers are by their nature extremely high cardinality objects.  Our flexible string search will match substrings in the container name, ID, or image fields.
+
+### Tagging
+
+Containers are tagged with all existing host-level tags.  We also tag with metadata associated with individual containers. 
+
+All containers are tagged by `#container_image`, and additionally, we include integrations with popular orchestrators, such as ECS and Kubernetes, which provide further container-level tags.  We also decorate each container with Docker, ECS, or Kubernetes icons so you can tell which are being orchestrated at a glance.
+
+ECS Containers are tagged by `#task_name`, `#task_version`, and `#ecs_cluster`
+
+Kubernetes Containers are tagged by `#pod`, `#pod_ip`, `#service`, `#namespace`, `#cluster-name`, `#replica_set`, `#daemon_set`, `#job`, and `#deployment`.
```

Comment:
worth changing to `kube_pod`, `kub_service`, .. etc?

Reply:
yes